packages/models/src/lib/implementation/schemas.ts

```diff
@@ -96,6 +101,9 @@ export function metaSchema(options?: {
       docsUrl: docsUrlDescription
         ? docsUrlSchema.describe(docsUrlDescription)
         : docsUrlSchema,
+      isSkipped: isSkippedDescription
+        ? isSkippedSchema.describe(isSkippedDescription)
+        : isSkippedSchema,
```

Comment:
Could you please also re-generate the Markdown docs (`npx nx generate-docs models`)?